pkg/controllers/cstorvolumeconfig/volume_operations.go

```diff
@@ -420,6 +420,11 @@ func (c *CVCController) createCVR(
 		annotations[string(apis.SnapshotNameKey)] = snapName
 		labels[string(apis.CloneEnableKEY)] = isClone
 	}
+	// Set isRestoreVol annotation on CVR if CVC has
+	// "openebs.io/created-through: restore" annotation
+	if value := claim.GetAnnotations()["openebs.io/created-through"]; value == "restore" {
+		annotations["isRestoreVol"] = "true"
+	}
```

Comment:
nit_: As of now, we can think of moving the string to a constant in the current file only.

Reply:
Also why the annotation key is `isRestoreVol` ? And not something like `cstor.openebs.io/isrestore`

Reply:
> Also why the annotation key is isRestoreVol ? And not something like cstor.openebs.io/isrestore

Followed the old convention we can change it.

Reply:
It can be taken as next PR.